src/style.css

```
@@ -1 +1,20 @@
 /* styles go here */
+.blok-input {
+  padding-bottom: 20px;
+}
+
+.input-content {
+  padding-bottom: 10px;
+}
+
+.intut-radio {
```

```suggestion
.input-radio {
```
Тут помилка в слові input. Але я все одно не бачу, де цей клас використовується в коді